operator/config/crd/bases/platform.stackrox.io_securedclusters.yaml

```diff
@@ -404,6 +404,7 @@ spec:
                         - EBPF
                         - CORE_BPF
                         - NoCollection
+                        - KernelModule
```

Comment:
Is it automatically switched to eBPF in code when a customer selects `KernelModule`?
I am worried that customer uses `KernelModule` and the feature was removed from ACS.

Reply:
Collector switches automatically to eBPF, I tested this.